test/run_test.py

```diff
@@ -560,6 +581,16 @@ def get_configurations(config_params, intraprocess, shm):
     if shm is not None and shm:
         flags = [f for f in flags if f[0] != 'SHM_OFF']
 
+    if security is not None:
+        props_file = open(config_params['properties']['SECURITY'], "r+")
+        data = props_file.read()
+        #Replace all occurrences of the required label
+        data = data.replace('<CERTS_RELATIVE_PATH>', args.certs_path)
```

Comment:
I think we should add some sanity check to test that the provided certs_path location is actually a valid directory before using it here.